docs/proposals/cluster-proxy/clusterproxy.md

```diff
@@ -0,0 +1,263 @@
+
+Also, Karmada is usually used in multi-tenant environment, user A cloud not access to the resource of user B. This is always achieved by using RBAC. Which means the query operation should follow the RBAC privilege limitations in member clusters.
+
+So this proposal proposes an enhancement to query resources from member clusters with native `kubectl`, unaware the member clusters, and follow RBAC privilege limitations in member clusters.
+
+With this enhancement, users can feel the minimal changes after migrating to Karmada, but enjoy the benefits of multiple clusters.
+
+### Goals
+
+- Propose the idea to query resources from multiple clusters with native `kubectl` and without needing to be aware of the individual member clusters.
+- Propose the implementation ideas for involved components.
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+### Non-Goals
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. What is the desired outcome and how do we measure success?.
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+### User Stories (Optional)
+
+<!--
+Detail the things that people will be able to do if this KEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+#### Story 1
+As a user, I want to migrate my workloads to multiple clusters using Karmada while experiencing minimal differences in the user interface. Specifically, I expect the following:
+- The ability to perform operations and maintenance tasks with `kubectl`, just like before.
+- No need to be aware of the member clusters.
+Therefore, I would like Karmada to provide a seamless and native `kubectl` experience.
+
+#### Story 2
+As a platform administrator, I provide a multi-cluster platform by using Karmada, and this platform is used by multiple users. So in order to limit the access range for different users, I configure the RBAC in different member clusters to achieve this. So I want that any operations from the users should follow the RBAC in member clusters.
+
+### Notes/Constraints/Caveats (Optional)
+
+<!--
+What are the caveats to the proposal?
+What are some important details that didn't come across above?
+Go in to as much detail as necessary here.
+This might be a good place to talk about core concepts and how they relate.
+-->
+
+- If there are resources with the same name in different member clusters, operations such as `get`, `describe`, `exec`, and `logs` will fail.
```

Comment:
How about add a external parameter like "--c clustername' to specify the cluster? If not point the multiple resource objects  confusion .